diff -aurN new_000/interface/vmcs_host/CMakeLists.txt org_000/interface/vmcs_host/CMakeLists.txt
--- new_000/interface/vmcs_host/CMakeLists.txt	2021-05-27 13:33:20.608632186 +0900
+++ org_000/interface/vmcs_host/CMakeLists.txt	2021-05-27 12:41:24.280125844 +0900
@@ -8,8 +8,8 @@
 add_definitions(-fno-strict-aliasing)
 
 add_library(vchostif
-            ${VMCS_TARGET}/vcmisc.c
-            vc_vchi_gencmd.c vc_vchi_gpuserv.c
+            ${VMCS_TARGET}/vcfilesys.c ${VMCS_TARGET}/vcmisc.c
+            vc_vchi_gencmd.c vc_vchi_filesys.c vc_vchi_gpuserv.c
             vc_vchi_tvservice.c vc_vchi_cecservice.c
             vc_vchi_dispmanx.c vc_service_common.c)
 #            ${VMCS_TARGET}/vmcs_main.c
