diff -aurN new_001/interface/vmcs_host/CMakeLists.txt new_002/interface/vmcs_host/CMakeLists.txt
--- new_001/interface/vmcs_host/CMakeLists.txt	2021-05-27 13:58:07.741637816 +0900
+++ new_002/interface/vmcs_host/CMakeLists.txt	2021-05-27 14:26:42.885982663 +0900
@@ -8,14 +8,22 @@
 add_definitions(-fno-strict-aliasing)
 
 set(VCHOSTIF_SOURCE
-    ${VMCS_TARGET}/vcfilesys.c ${VMCS_TARGET}/vcmisc.c
-    vc_vchi_gencmd.c vc_vchi_filesys.c vc_vchi_gpuserv.c
+    ${VMCS_TARGET}/vcmisc.c
+    vc_vchi_gencmd.c vc_vchi_gpuserv.c 
     vc_vchi_tvservice.c vc_vchi_cecservice.c
     vc_vchi_dispmanx.c vc_service_common.c)
 #            ${VMCS_TARGET}/vmcs_main.c
 #  vc_vchi_haud.c
 
-add_library(vchostif SHARED ${VCHOSTIF_SOURCE})
+if (BUILD_WAYLAND)
+wayland_add_protocol_server(
+    VCHOSTIF_SOURCE
+    ../../interface/wayland/dispmanx.xml
+    dispmanx
+)
+endif ()
+
+add_library(vchostif SHARED ${VCHOSTIF_SOURCE})
 
 #add_library(bufman            vc_vchi_bufman.c            )
 set(INSTALL_TARGETS vchostif)
